cpp/ps0008/problem. 8.cpp

```diff
@@ -38,11 +44,16 @@ int main() {
         } else {
             std::cout << "Wrong input, try again" << std::endl;
         }
-        }
+    }
+    
     std::cout << "Enter the encrypted word or sentence:\n  ";
-    std::cin.ignore(); 
-    std::getline(std::cin, input);
+    std::cin.ignore();
+    std::getline(std::cin, input); // Taking message 1/1
```

Comment:
- [x] prompted user for ciphertext (encrypted message)
- [x] stored the targeted ciphertext inside a string variable